### DISCUSSION TITLE
How to access minio console In Openshift after installing a minio tenant?

### DISCUSSION THREAD

**Discussion by vamseekrishna25** [minio/minio#11859] [Q&A]
How to access minio console In Openshift after installing a minio tenant using a url in cluster

**Comment by Alevsk** [comment]
Hi @vamseekrishna25 I'm not super familiar with openshift but you need to expose the tenant-console service, in plain kubernetes this can be done via ingress, LoadBalancer or NodePort service (you can also port-forward for development purposes)

https://docs.okd.io/3.11/minishift/openshift/exposing-services.html

**Reply by vamseekrishna25** [reply]
i have exposed the service of the tenant but getting this 
![image](https://user-images.githubusercontent.com/9658248/112198129-cec1e880-8c32-11eb-87c8-0b6da64523f8.png)
instead of a login page
when i expose console service  of minio operator i am getting a login page to enter jwt token
![image](https://user-images.githubusercontent.com/9658248/112198547-39732400-8c33-11eb-8d21-5bbb0afa0d14.png)
i am new to minio i created the minio tenant  in my cluster trying to access it in my nodejs application in minio operator readme i saw api documentation any example how it can be done

**Reply by debasisbalabantaray** [reply]
Hi @vamseekrishna25. Off the topic, could you please help me knowing how you have deployed MinIO in OpenShift. I am new to this world. And I have requirement to deploy MinIO in OpenShift.

Thanks,
Debasis

**Reply by vamseekrishna25** [reply]
hi @debasisbalabantaray 
you can use this minio operator for kubernetes to deploy minio in openshift, as openshift is built on kubernetes this operator works
https://github.com/minio/operator/blob/master/README.md
https://docs.min.io/minio/k8s/tutorials/deploy-minio-tenant.html
